src/Atmos/Parameterizations/CloudPhysics/Activation.jl

```diff
@@ -0,0 +1,154 @@
+
+
+Pkg.add("SpecialFunctions")
+module Activation
+
+using Thermodynamics
+using CLIMAParameters
+using CLIMAParameters.Atmos.Microphysics_0M
+const APS = AbstractParameterSet
+export activation_cloud_droplet_number
+# TODO: Check over these ^^^^
+
+"""
+    Isabella Dula and Shevali Kadakia
```

Comment:
The description of what the file does should be at the very top. Then `module` , `using` and `export` lines